<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-E3EDNBP" failures="1" tests="7" name="Test2" time="0.134" errors="0" timestamp="2024-03-12T17:56:41 IST">
  <testcase classname="testng.Basic2" name="daata" time="0.017"/>
  <testcase classname="testng.Basic2" name="daata" time="0.002"/>
  <testcase classname="testng.Basic2" name="daata" time="0.002"/>
  <testcase classname="testng.Basic2" name="test14" time="0.002"/>
  <testcase classname="testng.Basic2" name="fail" time="0.022">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testng.Basic2.fail(Basic2.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- fail -->
  <testcase classname="testng.Basic2" name="test15" time="0.006"/>
  <testcase classname="testng.Basic2" name="test13" time="0.002"/>
</testsuite> <!-- Test2 -->
